CONTRIBUTING.md

```diff
@@ -1,3 +1,31 @@
 # Contributing
 
 We follow the [IPython Contributing Guide](https://github.com/ipython/ipython/blob/master/CONTRIBUTING.md).
+
+## To set up a development environment
+
+Fork the repository and clone the forked repository locally.
+
+Use Conda to install dependencies and activate the development environment.
+
+```
+    conda create -n qtdev python=3
+    conda activate qtdev
+    conda install pyqt
+```
+Setup **qtconsole** to run on the development environment:
+
+```
+  python setup.py develop
+```
+
+To run after the changes have been made to source:
+```
+  pip install -e .
```

Comment:
What's the difference between `python setup.py develop` and `pip install -e .`? Why should we need to run both? (Hey, I'm learning something new about python, cool!)

Reply:
I don't think you need to run both. Maybe that should be reworded to clear out that confusion. I am not very sure about the difference, but I based my commit on what I understood by Carol's comments: https://github.com/jupyter/qtconsole/issues/302#issuecomment-427733177.

Please correct me if I am wrong.

Reply:
Seems like there are some subtle differences:
https://stackoverflow.com/questions/30306099/pip-install-editable-vs-python-setup-py-develop

Reply:
Ah, thanks for the link.